pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/entity/PatientRecordEntity.java

```diff
@@ -38,8 +38,12 @@
 	@Column(name = "ssn")
 	private String ssn;
 	
-	@Column(name = "gender")
-	private String gender;
+	@Column(name = "patient_gender_id")
+	private Long patientGenderId;
+	
+	@OneToOne
+	@JoinColumn(name="id")
```

Comment:
Does this work? I thought the JoinColumn would need to be patient_gender_id here as well